go.mod

```diff
@@ -108,19 +108,19 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.12.2
-	istio.io/api v1.19.0-rc.0.0.20230828012244-464da4944bb2
+	istio.io/api v1.19.0-beta.1.0.20230821193953-6d232ba686ad
```

Comment:
I think now this can be updated to `v1.19.0`?

Reply:
See https://github.com/istio/client-go/pull/1739

Reply:
Once that PR merges, the pipeline will push the change to istio/istio.

Reply:
Thanks for the info

Reply:
> Once that PR merges, the pipeline will push the change to istio/istio.

Or since the pipeline was held, we may find we have a rebase issue, and have to fix it manually. Will know more once these PRs start getting approved/un held.

Reply:
@ericvn I think one thing may not be correct is the `istio/api` repo's version is changed from `rc` to `beta`